modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18189,6 +18199,8 @@ the-friendly-url-of-the-page-you-are-trying-to-save-conflicts-with-a-friendly-ur
 the-friendly-url-of-the-site-template-page-you-are-trying-to-save-conflicts-with-some-of-the-own-pages-of-the-sites-created-from-this-site-template.-are-you-sure-you-want-to-configure-the-site-template-page-with-this-friendly-url=The friendly URL of the site template page you are trying to save conflicts with some of the own pages of the sites created from this site template. Are you sure you want to configure the site template page with this friendly URL?
 the-friendly-url-of-this-page-is-conflicting-with-a-friendly-url-of-a-page-in-the-site-template,-from-which-this-site-was-created=The friendly URL of this page is conflicting with a friendly URL of a page in the site template, from which this site was created:
 the-friendly-url-x-was-changed-to-x-to-ensure-uniqueness=The friendly URL {0} was changed to {1} to ensure uniqueness.
+the-frontend-token-definition-is-a-json=The frontend token definition is a JSON file that allows you to contribute your own tokens. It will replace the current tokens contributed by the theme.
```

Comment:
Hey @thiago-buarqque , I think Brian always wants the key to match the full string, right? Sorry I missed this before.

Reply:
Ops, did not know that. Thought we could do it for large messages.